curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
+Which positioning method allows an element to stick to a defined position only when you scroll past a certain point?
 
 #### --distractors--
 
-Placeholder distractor 1
+`position: float`
 
 ---
 
-Placeholder distractor 2
+`position: fixed`
 
 ---
 
-Placeholder distractor 3
+`position: absolute`
 
 #### --answer--
 
-Placeholder answer
+`position: sticky`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What would happen if an element had `z-index: -1`?
 
 #### --distractors--
 
-Placeholder distractor 1
+It will be on top of all the other elements on the page
 
 ---
 
-Placeholder distractor 2
+It will disappear from view.
 
 ---
 
-Placeholder distractor 3
+It will stack on top of all other elements
 
 #### --answer--
 
-Placeholder answer
+It will always be in the back of all the other elements on the page
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How does a floated element affect the layout of surrounding elements?
 
 #### --distractors--
 
-Placeholder distractor 1
+Other elements are placed in relation to the floated element
 
 ---
 
-Placeholder distractor 2
+The surrounding elements ignore the floated element and overlap it.
 
 ---
 
-Placeholder distractor 3
+It will shift surrounding elements behind the floated element
 
 #### --answer--
 
-Placeholder answer
+Block elements surrounding it flow around the floated element
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does an element with `position: sticky` require for it to work?
 
 #### --distractors--
 
-Placeholder distractor 1
+A `position: relative` parent element 
 
 ---
 
-Placeholder distractor 2
+A `z-index` value greater than 1
 
 ---
 
-Placeholder distractor 3
+A `z-index` of 0 or greater
 
 #### --answer--
 
-Placeholder answer
+A declared `top`, `left`, `right`, or `bottom` property 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when setting the `overflow` property of a parent to `auto` if one of its children has the property `position: sticky`?
 
 #### --distractors--
 
-Placeholder distractor 1
+The child element's sticky behaviour is not affected by the overflow property of the parent.
 
 ---
 
-Placeholder distractor 2
+The sticky element will maintain its behavior regardless of scrolling.
 
 ---
 
-Placeholder distractor 3
+The sticky element will act as if it had position: fixed within the parent.
 
 #### --answer--
 
-Placeholder answer
+The child will lose its sticky behaviour when the parent scrolls.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following are true about `position: relative` and how it affects the child elements?
 
 #### --distractors--
 
-Placeholder distractor 1
+Child elements with `position: absolute` will be positioned relative to the nearest positioned ancestor, whether that is relative or absolute.
 
 ---
 
-Placeholder distractor 2
-
+A `position: relative` element moved does not occupy any space within the document flow.
```

Comment:
```suggestion
A `position: relative` element moved does not occupy any space within the document flow.

```

Reply:
you still need to address this @Muhammad-Rebaal